dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -783,9 +783,9 @@
             "modelAsString": true
           }
         },
-        "documentType": {
+        "healthcareDocumentType": {
```

Comment:
I think we would like to keep the property name as `documentType` and update its type only.